src/controllers/auth.controller.js

```diff
@@ -25,13 +26,23 @@ const authController = (User) => {
     }));
   };
 
-  const register = (req, res) => {
-
-  };
+  const register =(req, res) => {
+      var user = new User();
+       user.set("name",req.body.name);
+       user.set("email", req.body.email);
+       user.set("password", req.body.password);
 
+       user.save().then((user)=> {
+          res.status(200).json({message:'User created successfully with name: ' + user.get("name") + 'and email: ' + user.get("email")});
+      }).catch((err)=>{
+        res.status(400).json({
+          error: err,
+      });
+  });
+}
```

Comment:
Your model usage is quite different from the structure we had, we have to maintain consistency,  remember the user model was injected into the auth controller through the parameter. have a look at sequelize doc here https://sequelize.org/v5/manual/getting-started.html. to insert we use .create, so in this case, you should do a User.create({}).then().catch()